share/eds/blockstore.go

```diff
@@ -0,0 +1,210 @@
+package eds
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+	"sync"
+
+	"github.com/filecoin-project/dagstore"
+	"github.com/filecoin-project/dagstore/shard"
+	lru "github.com/hashicorp/golang-lru"
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	ipld "github.com/ipfs/go-ipld-format"
+)
+
+var _ blockstore.Blockstore = (*Blockstore)(nil)
+
+var (
+	maxCacheSize            = 100
```

Comment:
Just curious, how this value is chosen? 
Also u
sually preallocated buffer sizes tends to be power of 2, because it fits memory layout better.

Reply:
Yeah, don't know why I picked 100, changed to 128, thanks!